Modules/Loadable/CornerText/qSlicerCornerTextDICOMAnnotationPropertyValueProvider.cxx

```diff
@@ -0,0 +1,157 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#include <vtkMRMLSliceViewDisplayableManagerFactory.h>
+
+// Slicer includes
+#include <vtkSlicerApplicationLogic.h>
+#include <qSlicerApplication.h>
+#include <qSlicerPythonManager.h>
+
+// CornerText includes
+#include "qSlicerCornerTextDICOMAnnotationPropertyValueProvider.h"
+
+// DisplayableManager initialization
+#include <vtkAutoInit.h>
```

Comment:
```suggestion
```

This should only be includes from  `qSlicerCornerTextModule.cxx` where I would also expect the macro `VTK_MODULE_INIT` to also be used.